From 808601f075fe3dd8ca101e31c9121a4e87f0bfc7 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Thu, 6 May 2021 07:45:00 +0200
Subject: [PATCH] gdb/configure.tgt: build gdbserver with m68k and
 uclibc

Allow to build gdbserver with m68k and uclibc:

checking whether gdbserver is supported on this host... no
configure: error: Automatic gdbserver build is not supported for this configuration

Fixes:
 - http://autobuild.buildroot.org/results/f4d6d9d8418c0da48a3db4ad5a82e19bd16eae34

Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 gdb/configure.tgt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gdb/configure.tgt b/gdb/configure.tgt
index caa42be1c01..8666bc5d0bf 100644
--- a/gdb/configure.tgt
+++ b/gdb/configure.tgt
@@ -372,12 +372,12 @@ m68hc11*-*-*|m6811*-*-*)
 	gdb_sim=../sim/m68hc11/libsim.a
 	;;
 
-m68*-*-aout* | m68*-*-coff* | m68*-*-elf* | m68*-*-rtems* | m68*-*-uclinux* | \
+m68*-*-aout* | m68*-*-coff* | m68*-*-elf* | m68*-*-rtems* | \
 fido-*-elf*)
 	# Target: Motorola m68k embedded
 	gdb_target_obs="m68k-tdep.o"
 	;;
-m68*-*-linux*)
+m68*-*-*linux*)
 	# Target: Motorola m68k with a.out and ELF
 	gdb_target_obs="m68k-tdep.o m68k-linux-tdep.o solib-svr4.o \
 			linux-tdep.o glibc-tdep.o symfile-mem.o"
-- 
2.30.2

